src/components/Pages/Guide/index.js

```diff
@@ -30,7 +30,6 @@ function Guide({ guidePage, intl }) {
       topics,
       theme,
       department,
-      relatedDepartments,
```

Comment:
this is the only line I changed in this file, there were no other references to it in there

the rest is just jsprettier